src/utils/components/AddBootableVolumeModal/components/SourceTypeSelection/SourceTypeSelection.tsx

```diff
@@ -22,6 +29,12 @@ const SourceTypeSelection: FC<SourceTypeSelectionProps> = ({ formSelection, setF
     [setFormSelection],
   );
 
+  useEffect(() => {
+    if (!loading && !canCreatePVC) {
+      setFormSelection(DROPDOWN_FORM_SELECTION.USE_REGISTRY);
+    }
+  }, [canCreatePVC, loading, setFormSelection]);
+
```

Comment:
should we also make same check for canCreateDS?
I think maybe it's best the get the useCanCreateBootableVolume values as props and this way on initial render you would have a correct display of what is disabled what is not

Reply:
I make this check just because the select has UPLOAD as default value. 
So only in that case we need to change it.